fhir-parent/checkstyle.xml

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE module PUBLIC
   "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
```

Comment:
should this change too or its still called "Puppy Crawl" here?  not too familiar with DOCTYPEs...

Reply:
updated `"-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"`
in next commit